src/widgets/NoWidgetEventTrait.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace hipanel\widgets;
+
+use yii\base\WidgetEvent;
+
+trait NoWidgetEventTrait
+{
+    public function init()
+    {
+    }
```

Comment:
```suggestion
    public function trigger($name, Event $event = null)
    {
        $excludedEvents = [self::EVENT_INIT];
        
        if (in_array($name, $excludeEvents, true)) {
            return;
        }
        
        parent::trigger($name, $event);
    }
```

В некоторых виджетах `init()` используется.
Возможно, лучше так?
Тогда, возможно, и нижестоящие методы можно будет дропнуть?